
{'date': 'Wed Sep 12 2018 10:13:02.881 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 2
    from _spy.tesouro.main import main
  module _spy.tesouro.main line 32
    STYLE["width"] = 800
AttributeError: 'module' object has no attribute '__setitem__'
'''},
{'date': 'Wed Sep 12 2018 11:18:10.200 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 2
    from _spy.tesouro.main import main
ImportError: cannot import name 'main'
'''},
{'date': 'Wed Sep 12 2018 11:22:15.710 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 2
    from _spy.tesouro.main import main
  module _spy.tesouro.main line 32
    STYLE["width"] = 800
AttributeError: 'module' object has no attribute '__setitem__'
'''},
{'date': 'Wed Sep 12 2018 11:25:29.630 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 3
    main()
  module _spy.tesouro.main line 215
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
TypeError: 'module' object is not callable
'''},
{'date': 'Wed Sep 12 2018 11:27:42.595 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
Exception: unexpected token: '.'
  module '$exec_459' line 3
main()
'''},
{'date': 'Wed Sep 12 2018 11:34:22.468 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
Exception: unexpected token: '.'
  module '$exec_1854' line 4
main(gui=Gui())
'''},
{'date': 'Wed Sep 12 2018 11:38:51.212 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 6
    def main(jogadores=JOGADORES, gui=GUI):
NameError: name 'JOGADORES' is not defined
'''},
{'date': 'Wed Sep 12 2018 11:39:42.909 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 28
    main(gui=Gui()).inicia()
  module <module> line 25
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 10
    shuffle(_chance)
NameError: name 'shuffle' is not defined
'''},
{'date': 'Wed Sep 12 2018 11:49:16.807 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 26
    gui.inicia()
AttributeError: 'Gui' object has no attribute 'inicia'
'''},
{'date': 'Thu Sep 13 2018 08:05:48.36 GMt-0300 (-03) -X- SuPyGirls -X-',
'error': '''
 module roxanne.main line 20
  cautela self.cartas > 9
           ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 09:40:52.710 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:41:46.13 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:41:39.716 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:41:26.383 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:42:05.632 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:42:02.12 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:41:39.722 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:42:14.614 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:42:29.457 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:41:53.771 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:41:55.960 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:41:58.648 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:42:01.383 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:42:04.182 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:42:47.360 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:42:16.944 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:42:19.56 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:42:21.524 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:45:15.710 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:45:51.412 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:45:58.250 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:46:37.806 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:47:26.375 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module angie.main line 10
  self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\ 
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 09:47:38.195 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module angie.main line 10
  self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\ 
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 09:47:50.74 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module angie.main line 10
  self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\ 
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 09:48:14.742 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module angie.main line 10
  self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\ 
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 09:48:04.652 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module angie.main line 10
  self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\ 
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 09:48:35.721 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module angie.main line 10
  self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\ 
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 09:48:38.67 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:49:01.80 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module angie.main line 10
  self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\ 
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 09:49:02.536 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module angie.main line 10
  self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\ 
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 09:49:24.546 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module angie.main line 10
  self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\ 
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 09:51:46.169 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:52:55.185 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module angie.main line 10
  self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\ 
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 09:52:42.953 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module angie.main line 10
  self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\ 
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 09:53:11.133 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module angie.main line 10
  self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\ 
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 09:53:39.76 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogado278
NameError: name 'Jogado278' is not defined
'''},
{'date': 'Thu Sep 20 2018 09:54:50.552 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module angie.main line 10
  self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\ 
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 09:56:05.941 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module angie.main line 10
  self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\ 
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 09:56:59.796 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 10
  _chance = list(range(20)
                                        ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 09:57:27.560 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module <string> line 15
  def _inicia(self, jogador)):
                            ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 09:59:37.773 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 20 2018 10:00:01.378 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 20 2018 10:00:55.432 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from sara.main import sara; self.joga = sara().joga
  module sara.main line 27
    return medo()
NameError: name 'medo' is not defined
'''},
{'date': 'Thu Sep 20 2018 10:01:57.122 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from sara.main import sara; self.joga = sara().joga
  module sara.main line 27
    return medo()
NameError: name 'medo' is not defined
'''},
{'date': 'Thu Sep 20 2018 10:03:50.463 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 20 2018 10:03:27.229 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 20 2018 10:04:09.90 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 20 2018 10:04:30.958 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 20 2018 10:07:07.35 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 20 2018 10:07:34.339 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 20 2018 10:07:41.178 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 20 2018 10:08:03.342 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 20 2018 10:10:16.62 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 20 2018 10:12:04.719 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module angie.main line 10
  self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\ 
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 10:14:37.672 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module angie.main line 10
  self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\ 
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 20 2018 10:20:00.290 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''
 module angie.main line 10
  self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\ 
                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 27 2018 09:39:50.22 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 27 2018 09:39:54.567 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 27 2018 09:40:30.241 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 27 2018 09:41:15.920 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 27 2018 09:41:28.426 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 27 2018 09:41:37.555 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 27 2018 09:41:50.584 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 27 2018 09:42:57.437 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 27 2018 09:43:06.656 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 27 2018 09:42:54.791 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 27 2018 09:42:56.539 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 27 2018 09:43:17.361 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 27 2018 09:45:03.359 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from courtney.main import courtney; self.joga = courtney().joga
  module courtney.main line 28
    return Jogador()
  module courtney.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: too many values to unpack (expected 6)
'''},
{'date': 'Thu Sep 27 2018 09:47:41.285 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from tracy.main import tracy; self.joga = tracy().joga
  module tracy.main line 28
    return Jogador()
  module tracy.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: need more than 5 values to unpack
'''},
{'date': 'Thu Sep 27 2018 09:48:03.294 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from tracy.main import tracy; self.joga = tracy().joga
  module tracy.main line 28
    return Jogador()
  module tracy.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: need more than 5 values to unpack
'''},
{'date': 'Thu Sep 27 2018 09:48:13.125 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from tracy.main import tracy; self.joga = tracy().joga
  module tracy.main line 28
    return Jogador()
  module tracy.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: need more than 5 values to unpack
'''},
{'date': 'Thu Sep 27 2018 09:49:16.92 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from tracy.main import tracy; self.joga = tracy().joga
  module tracy.main line 28
    return Jogador()
  module tracy.main line 8
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: need more than 5 values to unpack
'''},
{'date': 'Thu Sep 27 2018 09:50:24.908 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from tracy.main import tracy; self.joga = tracy().joga
  module tracy.main line 29
    return Jogador()
  module tracy.main line 9
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: need more than 5 values to unpack
'''},
{'date': 'Thu Sep 27 2018 09:50:12.808 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from tracy.main import tracy; self.joga = tracy().joga
  module tracy.main line 29
    return Jogador()
  module tracy.main line 9
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: need more than 5 values to unpack
'''},
{'date': 'Thu Sep 27 2018 09:50:55.868 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from tracy.main import tracy; self.joga = tracy().joga
  module tracy.main line 29
    return Jogador()
  module tracy.main line 9
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: need more than 5 values to unpack
'''},
{'date': 'Thu Sep 27 2018 09:51:18.799 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from tracy.main import tracy; self.joga = tracy().joga
  module tracy.main line 29
    return Jogador()
  module tracy.main line 9
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: need more than 5 values to unpack
'''},
{'date': 'Thu Sep 27 2018 09:51:27.892 GMt-0300 (Hora oficial do Brasil) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    main(gui=Gui()).inicia()
  module <module> line 27
    return Jogo(gui=gui, jogadores=[JogadorSimples(None, jogador) for jogador in jogadores])
  module <module> line 13
    self._inicia(nome)
  module <module> line 20
    except (ImportError, TypeError):
  module <module> line 1
    from tracy.main import tracy; self.joga = tracy().joga
  module tracy.main line 29
    return Jogador()
  module tracy.main line 9
    self.perigos, self.artefatos, self.cartas, self.rodada_corrente,\
ValueError: need more than 5 values to unpack
'''},